application/controllers/admin/ParticipantsAction.php

```diff
@@ -2553,7 +2553,7 @@ public function addToTokenattmap()
 
         // TODO: This code can't be reached
         echo "<p>";
-        printf(gT("%s participants have been copied to the survey survey participants table"), "<span class='badge rounded-pill bg-success'>" . $response['success'] . "</span>");
+        printf(gT("%s participants have been copied to the survey survey participants list"), "<span class='badge rounded-pill bg-success'>" . $response['success'] . "</span>");
```

Comment:
Can you fix the 'survey survey' please while you change the source string anyway ;)

Reply:
Applied the change